README.md

```diff
@@ -20,20 +20,21 @@ yarn add toucan-sdk
 
 # Quickstart
 
-Instantiate the Toucan client to interact with our infrastructure.
+Instantiate the ContractInteractions and the SubgraphInteractions clients to interact with our infrastructure.
 
 ```typescript
-import ToucanClient from "toucan-sdk";
```

Comment:
I'd recommend we start with an example that uses `ToucanClient` and then have separate examples using the inner clients while explaining **why** would a developer choose to use an inner client vs the main client.